ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateStreamCommandTest.java

```diff
@@ -134,6 +140,21 @@ public void shouldThrowOnOldWindowProperty() {
     createCmd();
   }
 
+  @Test
+  public void testCreateAlreadyRegisteredStreamThrowsException() {
+    // Given:
+    givenProperties(propsWith(ImmutableMap.of()));
+
+    // When:
+    final CreateStreamCommand cmd = createCmd();
```

Comment:
It's worth explaining the Given/When/Then thing in case you're not familiar.

Positive tests:

- Given: is the setup for the test, i.e. setting up the pre-conditions.
- When: is the actual _line_ on test.
- Then: is asserting the post conditions.

Negative tests:

- Given: is the setup for the test, i.e. setting up the pre-conditions.
- Expect/Then: the execpted exceptions
- When: is the actual _line_ on test.

So in this test might look like:

```java
// Given:
givenProperties(..)
cmd = createCmd()
cmd.run();

// Expect:
expectedException.blah

// When:
cmd.run();

Reply:
Thanks, I'll use that way